rules/windows/network_connection/sysmon_susp_prog_location_network_connection.yml

```diff
@@ -12,19 +12,19 @@ logsource:
     definition: 'Use the following config to generate the necessary Event ID 3 Network Connection events'
 detection:
     selection:
-        Image: 
-            # - '*\ProgramData\\*'  # too many false positives, e.g. with Webex for Windows
-            - '*\$Recycle.bin'
-            - '*\Users\All Users\\*'
-            - '*\Users\Default\\*'
-            - '*\Users\Public\\*'
-            - '*\Users\Contacts\\*'
-            - '*\Users\Searches\\*' 
-            - 'C:\Perflogs\\*'
-            - '*\config\systemprofile\\*'
-            - '*\Windows\Fonts\\*'
-            - '*\Windows\IME\\*'
-            - '*\Windows\addins\\*'
+        Image|contains: 
+            # - '\ProgramData\\'  # too many false positives, e.g. with Webex for Windows
+            - '\$Recycle.bin'
+            - '\Users\All Users\\'
+            - '\Users\Default\\'
+            - '\Users\Public\\'
+            - '\Users\Contacts\\'
+            - '\Users\Searches\\' 
+            - 'C:\Perflogs\\'
```

Comment:
This one was `startswith` in the original rule (:

Reply:
Fixed :)